core/src/main/java/com/confighub/core/resolver/Context.java

```diff
@@ -135,6 +140,32 @@ protected boolean containsLevelAtDepth(Depth d, String levelName)
         return false;
     }
 
+    protected String getRegexPattern() {
```

Comment:
is this only for the db pattern, may want to specify a more specific name

Reply:
Changed in latest version